docs/config.rst

```diff
@@ -39,7 +39,7 @@ Any configuration key/value listed here may also be used in the specific plugin
 
     To learn more about GCP projects, please see `Google's docs on creating & managing projects <https://cloud.google.com/resource-manager/docs/creating-managing-projects>`_.
 
-.. option:: scopes=["STR", "STR"]
+.. option:: scopes=["STR","STR"]
```

Comment:
Sphinx was complaining about this when other options specified the same option (eg. `crm_scopes=["STR", "STR"]`). When the space is removed, other `option` statements can specify the same type attributes.